R/moveapps.R

```diff
@@ -156,15 +156,20 @@ createMoveAppsShinyUI <- function(request) {
     includeScript(system.file("shiny-apps/www/ws-keep-alive-fix.js", package = "moveapps")),
     includeCSS(system.file("shiny-apps/www/ws-keep-alive-fix.css", package = "moveapps")),
     includeScript(system.file("shiny-apps/www/extract-inputs.js", package = "moveapps")),
-
-    # functions (`shinyModuleUserInterface()` and `shinyModule()`) provided by the app developer
-    shinyModuleUserInterface("shinyModule"),
-
+    
     # ws-heartbeat fix
     # kudos: https://github.com/rstudio/shiny/issues/2110#issuecomment-419971302
     textOutput("ws_heartbeat"),
+    
     # store the current state (as a shiny bookmark)
-    bookmarkButton(id = 'ma_bookmark', label="Store settings", title="Click here to store the current chosen settings for future runs of the workflow",class="btn btn-outline-success"),
+    tags$div(
+      style = "position: relative; display: flex; justify-content: space-between; align-items: center; padding: 15px 20px;  border-bottom: none; margin-bottom: 0;",
+      h2(""),
+      bookmarkButton(id = 'ma_bookmark', label="Store settings", title="Click here to store the current chosen settings for future runs of the workflow",class="btn btn-outline-success", style = "margin: 0;")
+    ),
```

Comment:
Wenn es "nur" darum geht dass der Button rechtsbündig ist reicht dieser Style - das dummy element `h2` kann dadurch auch entfallen. 
Außerdem wirkt der Button "verloren" wenn extra padding gesetzt ist - hiermit ist er zusätzlich an der page ausgerichtet.

```suggestion
    tags$div(
      style = "display: flex; justify-content: flex-end;",
      bookmarkButton(id = 'ma_bookmark', label="Store settings", title="Click here to store the current chosen settings for future runs of the workflow",class="btn btn-outline-success", style = "margin: 0;")
    ),
```

Davor:
<img width="1068" height="354" alt="image" src="https://github.com/user-attachments/assets/7abd10a9-ffcd-467d-b374-843e648bafbd" />

Danach:
<img width="1062" height="296" alt="image" src="https://github.com/user-attachments/assets/721c552e-de56-4ef1-a282-31b2e90296d3" />

Reply:
Danke @cleemansen. Das mit dem h2("") als dummy habe ich tatsächlich vergessen nochmal zu überprüfen in der letzten runde. Davor immer wenn ich es rausgenommen habe ist der button nach links gesprungen. Total ärgerlich. 
Danke für das anpassen vom `style`. Zugegeben bastel ich Sachen zusammen mit wenig wissen, nur mit dem was mir vorgeschlagen wird, bis es halt das macht wie ich es mir in etwa vorstelle.